batch/test-locally.sh

```diff
@@ -23,3 +23,4 @@ do
 done
 
 POD_IP='127.0.0.1' BATCH_URL='http://127.0.0.1:5000' python -m unittest -v test/test_batch.py
+BATCH_USE_KUBE_CONFIG=1 python -m unittest -v test/test_batch_server.py
```

Comment:
Let's change the previous line to `test/test_*.py` instead. We should avoid setting `BATCH_USE_KUBE_CONFIG` directly because the `Makefile` sets it differently for different targets. `make test-local-in-cluster` in particular uses the latent k8s privileges of the pod to test locally on the pod.

Reply:
Done